app/controllers/users/settings/teams_controller.rb

```diff
@@ -134,6 +137,10 @@ def destroy
 
       private
 
+      def check_create_team_permission
+        can_create_teams?
```

Comment:
You probably want to do something if user can't create teams, like render 403 or something?